core/src/CMakeLists.txt

```diff
@@ -17,21 +17,10 @@ file(GLOB ledger-core-jni-sources
 
 if (TARGET_JNI)
     message(STATUS "Configure project for JNI")
-#    if(ANDROID)
-#        if (UNIX)
-#            set( JAVA_AWT_INCLUDE_PATH $ENV{JAVA_HOME}/include CACHE PATH "" FORCE)
-#            list(APPEND JAVA_INCLUDE_PATH2 $ENV{JAVA_HOME}/include)
-#            if(ANDROID_OSX)
-#                list(APPEND JAVA_INCLUDE_PATH2 $ENV{JAVA_HOME}/include/darwin)
-#                list(APPEND JAVA_AWT_LIBRARY $ENV{JAVA_HOME}/jre/lib/libjawt.dylib)
-#                list(APPEND JAVA_JVM_LIBRARY $ENV{JAVA_HOME}/jre/lib/server/libjvm.dylib)
-#            else()
-#                list(APPEND JAVA_INCLUDE_PATH2 $ENV{JAVA_HOME}/include/linux)
-#                list(APPEND JAVA_AWT_LIBRARY $ENV{JAVA_HOME}/jre/lib/i386/libjawt.so)
-#                list(APPEND JAVA_JVM_LIBRARY $ENV{JAVA_HOME}/jre/lib/i386/server/libjvm.so)
-#            endif()
-#        endif ()
-#    endif()
+    if (APPLE)
+        list(APPEND JAVA_INCLUDE_PATH2 $ENV{JAVA_HOME}/include/darwin)
```

Comment:
Discussion we had: you don't normally need this !

Reply:
Yep we don’t; I’m removing it.